docs/en/stack/ml/df-analytics/ml-dfa-overview.asciidoc

```diff
@@ -3,12 +3,14 @@
 == Overview
 
 {dfanalytics-cap} enable you to perform different analyses of your data and 
-annotate it with the results. Essentially, as part of its output, {dfanalytics} 
-appends the results of the analysis to the source data. By doing this, it 
-provides additional insights into the data. The process leaves the source index 
-intact, it creates a new index that contains a copy of the source data and the 
-annotated data. You can slice and dice the data extended with the results as you 
-normally do with any other data set.
+annotate it with the results. By doing this, it provides additional insights 
+into the data. {oldetection-cap} identifies unusual data points in the dataset. 
```

Comment:
I think it might be nice to turn these into links:
```suggestion
into the data. <<dfa-outlier-detection,{oldetection-cap}>> identifies unusual data points in the dataset. 
```